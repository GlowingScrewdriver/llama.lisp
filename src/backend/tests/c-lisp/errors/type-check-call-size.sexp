(c-lisp
    (define ((print int) (n int)))

    (define ((func_print void) (n int) (m int)) 
        (call print n)
        (call print m))

    (define ((main void)) 
        (declare k int)
        (declare j int)
        (declare m int)
        (set k 50)
        (set j 20)
        (set m 100)
        (call func_print k j m)))