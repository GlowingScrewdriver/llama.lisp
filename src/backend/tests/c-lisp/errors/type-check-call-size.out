Error in statement: ['call', 'func_print', 'k', 'j', 'm']
CodegenError: Expected types: ['int', 'int'], Received: ['int', 'int', 'int']
